src/vendors/peerly/services/peerlyIdentity.service.ts

```diff
@@ -356,6 +367,15 @@ export class PeerlyIdentityService extends PeerlyBaseConfig {
       postalCode: postalCode?.long_name,
       website: websiteDomain.substring(0, 100), // Limit to 100 characters per Peerly API docs
       email: `info@${domain.name}`.substring(0, 100), // Limit to 100 characters per Peerly API docs
+      jobAreas:
+        stateCode && areaCodes && areaCodes.length > 0
+          ? [
+              {
+                didState: stateCode,
+                disNpaSubset: areaCodes,
+              },
+            ]
+          : undefined,
```

Comment:
the Peerly API is rather finicky w/r/t what is passed in request bodies.

Let's just do a conditional key instead of sending an explicit `undefined` on requests.
```suggestion
...(stateCode && areaCodes && areaCodes.length > 0
        ? {
            jobAreas: [
              {
                didState: stateCode,
                disNpaSubset: areaCodes,
              },
            ],
          }
        : {}),
```

Reply:
changed